docs/api/web-frame.md

```diff
@@ -65,6 +65,15 @@ Sets the maximum and minimum layout-based (i.e. non-visual) zoom level.
 
 ### `webFrame.setSpellCheckProvider(language, provider)`
 
+* `language` String[]
+* `provider` Object
+  * `spellCheck` Function
+    * `words` Record<String, String[]>
+    * `callback` Function
+      * `misspeltWords` String[]
+
+### `webFrame.setSpellCheckProvider(language, provider)`
```

Comment:
Not sure what would be recommended way to create overload definition. Specifying union type (`language: String | String[]` and `words: String[] | Record<String, String[]>`) cannot narrow down param types, especially for `words` param `String[] | Record<String, String[]>` does not have interesection it's force required to cast types (which could be breaking changes in terms of types).

To make this work, listed each definition separately to create overloaded fn signatures.